signing_service/setup.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import os
+from setuptools import setup
+
+
+def get_version():
+    with open(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'RELEASE-VERSION',
+        ),
+        'r'
+    ) as version_file:
+        return version_file.read()
+
+
+setup(
+    name='Signing-Service',
+    version=get_version(),
+    url='',
+    maintainer='',
+    maintainer_email='',
+    packages=[
+        'signing_service',
+    ],
```

Comment:
What about a script to run the service? How do you run it when it's installed from a package?

Reply:
It still works after fixing, :+1: done